ui/src/app/views/initial-error/component.ts

```diff
@@ -4,39 +4,38 @@ import {NavigationError, Router} from '@angular/router';
 
 // Mostly borrowed from the Job Manager.
 @Component({
-    selector: 'app-not-found',
+    selector: 'app-initial-error',
     templateUrl: './component.html',
     styleUrls: ['./component.css']
 })
-@Injectable()
-export class NotFoundComponent {
+export class InitialErrorComponent {
   initialLoadErrorHeader: string;
   initialLoadFailure = false;
 
   constructor(router: Router, private loc: Location) {
```

Comment:
Odd - `loc` is private but the compiler doesn't complain about accessing it directly from the template.